### PR TITLE
docs: add Walrus Sites testnet docs

### PR DESCRIPTION
**Pull request by giac-mysten**

### CODE REVIEW

**Comment by giac-mysten** [comment]
@Tzal3x want to have a first look :) let me know if I forgot anything!

File: docs/walrus-sites/routing.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+```
+
+### Default headers
+
+By default, no headers need to be specified, and the `ws-resources.json` file can be omitted. The
+site-builder will automatically try to infer the `Content-Type` header based on the file extension,
+and set the `Content-Encoding` to `identity` (no transformation).
+
+In case the content type cannot be inferred, the `Content-Type` will be set to
+`application/octet-stream`.
```
Comment (Tzal3x) [inline docs/walrus-sites/routing.md:66]: Still skeptical about describing implementation details here. 

This is not something that will change anytime soon so it's fine I guess.

**Review by Tzal3x — APPROVED** [review]
Amazing! Very well put 💯

File: docs/walrus-sites/routing.md (hunk 1 above)
Comment (giac-mysten) [reply docs/walrus-sites/routing.md:66]: i think it's useful for devs that use weird extensions and then wonder why they have `octet-stream` headers coming in

File: docs/walrus-sites/commands.md
[the remark this answers is not in the document]
Comment (giac-mysten) [reply docs/walrus-sites/commands.md:1]: mmm I do agree in general. Here the goal is however to give the high level picture of the things that are not directly said in the `--help` output, such as the fact that testnet `1 epoch == 1 day`, so you may want to publish your site for longer, etc.

**Review by mlegner — APPROVED** [review]
Thank you very much, I love the new features and the docs! 🥇 

I made some very minor changes in a545419b85d44f0339df5b5bc2568e637be03cc5 and made "portal" lowercase as discussed offline.

**Pull request merged**